packages/wrangler/src/dev.tsx

```diff
@@ -354,7 +379,7 @@ function useTmpDir(): string | undefined {
     return () => {
       dir.cleanup().catch(() => {
         // extremely unlikely,
-        // but it's 2021 after all
+        // but it's 2022 after all
```

Comment:
🥲

Reply:
That should keep us busy over the next few years

Reply:
it's why they pay us the big bucks